CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [2.0.0] - 2018-04-24
```

Comment:
I have a question regarding the versioning scheme to use here. Probably more a question for @RobinL 

I'm just starting to have a look at this, so apologies if what I'm saying doesn't make any sense.

OK, so my understanding is that we'll pin the version of R the language (which is the most likely cause of incompatibility problems) and use the latest version of RStudio the IDE on top of that (don't know if we pin the version of it as well, didn't see the `Dockerfile` yet).

Now, few comments on versioning scheme:
1. Wouldn't be better to have our RStudio tag/version to follow RStudio version? I think it would. The reason being that this way anyone can look at our RStudio tag/version and know that that image is using RStudio version x (`3.4.2` in this case)
2. Bearing in mind in the future we may want users to see/pick the version of a tool to use
3. Probably the version or RStudio is less relevant. If we think it's important, should this also be part of the version/tag (probably overkill)
4. We obviously want to version our flavour of the R/RStudio combination so we should add something on top of the R version, this could be a sequential number

What I'm trying to get to here is, would a version/tag looking like `3.4.2-{OUR_SEQUENCIAL_NUMBER}` a bad versioning scheme/idea?

Reply:
Is there a problem with it being a bit more verbose?  I wonder whether something like r_base_3.4.2-moj_v1 could be a little more descriptive

Reply:
In theory tags can really do anything, it's just a string. But obviously it's better if they have a meaning/they're consistent otherwise we would be back to square one :P 

Maybe `r-3.4.2_moj-1.0.0`? Short answer is, yes it can be more descriptive (and maybe it should).